src/AppInstallerCommonCore/GroupPolicy.cpp

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "winget/GroupPolicy.h"
+#include "AppInstallerLogging.h"
+
+using namespace AppInstaller::StringResource;
+
+namespace AppInstaller::Settings
+{
+    namespace
+    {
+        GroupPolicy& InstanceInternal(std::optional<GroupPolicy*> overridePolicy = {})
+        {
+            // TODO: Use final location
+            constexpr std::string_view PoliciesKeyPath = "test\\policy"; // "SOFTWARE\\Policies\\Microsoft\\Windows\\WinGet";
+            static GroupPolicy s_groupPolicy{ Registry::Key::OpenIfExists(HKEY_CURRENT_USER /* HKEY_LOCAL_MACHINE */, PoliciesKeyPath) };

```

Comment:
I don't like checking this in with test values here; we should disable it until we have the real location.  I assume that constructing a default `GroupPolicy` would do that.